sorbet.html.markdown

```diff
@@ -0,0 +1,1040 @@
+---
+language: sorbet
+filename: learnsorbet.rb
+contributors:
```

Comment:
Maybe this should be tagged as a tool

Reply:
TypeScript is listed as a language, and Sorbet the same concept, but for Ruby instead of JavaScript.

Reply:
It's a bit different because Sorbet code remains valid Ruby code, but sure, let's go with that.